website/static/js/projectorganizer.js

```diff
@@ -1029,10 +1039,10 @@ function canAcceptDrop(items, folder) {
     if (hasFolders) {
         canDrop = canDrop && folder.data.permissions.acceptsFolders;
     }
-    if (copyMode === 'move') {
+    if (theCopyMode === 'move') {
```

Comment:
We talked, and it seemed not worth changing at this point, but I will avoid the construction in the future.